Makefile

```diff
@@ -16,29 +23,96 @@ osrm:
 phpunit:
 	@docker compose exec php php bin/console doctrine:schema:update --env=test --force --no-interaction --quiet
 	@docker compose exec php php vendor/bin/phpunit
+# Add as annotation in any testcase:
+# /**
+# * @group only
+# */
+# public function testSomething()..
+phpunit-only:
+	@clear && docker compose exec php php vendor/bin/phpunit --group only
```

Comment:
Not sure this belongs here... or maybe rename it `phpunit-debug` and use `@group debug` annotation?
There are other ways to execute only some tests, like 

```
vendor/bin/phpunit <filename>
vendor/bin/phpunit <filename> --filter=<testSomeMethodName>
```